model/team_search.go

```diff
@@ -9,9 +9,12 @@ import (
 )
 
 type TeamSearch struct {
-	Term    string `json:"term"`
-	Page    *int   `json:"page,omitempty"`
-	PerPage *int   `json:"per_page,omitempty"`
+	Term                    string `json:"term"`
+	Page                    *int   `json:"page,omitempty"`
+	PerPage                 *int   `json:"per_page,omitempty"`
+	AllowOpenInvite         *bool  `json:"allow_open_invite,omitempty"`
+	GroupConstrained        *bool  `json:"group_constrained,omitempty"`
+	IncludeGroupConstrained *bool  `json:"include_group_constrained,omitempty"`
```

Comment:
Will the fact that `IncludeGroupConstrained` has a default value of `false` cause calls to `(*TeamStore).SearchAll` and `(*TeamStore).SearchAllPaged` to exclude groups that used to be included?

Reply:
No the way i've set this up is that it `IncludeGroupConstrained` is only applied as a true value and when it is not set (nil) the query remains unchanged. If someone wants to exclude group constrained groups all they have to do is `GroupConstrained: false`

Reply:
Actually now that I re-think it it probably makes more sense to make this `ExcludeGroupConstrained`... which by default returns groups and if you set it to true it excludes group constrained results... that way the true and false value do exactly whats expected